applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,84 @@
+[id="odc-connecting-service-to-application-using-the-developer-perspective"]
+= Connecting a service to an application using the Developer perspective
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: odc-connecting-service-to-application-using-the-developer-perspective
+
+In addition to grouping multiple components within an application, you can also use the *Topology* view to connect components with each other. You can either use a binding connector or a visual one to connect components.
+
+A binding connection between the components can be established only if the target node is an Operator-backed service. This is indicated by the *Create a binding connector* tool-tip which appears when you drag an arrow to such a target node. When an application is connected to a service using a binding connector a `ServiceBinding` resource is created. Then, the Service Binding Operator controller projects the necessary binding data into the application deployment. After the request is successful, the application is redeployed establishing an interaction between the connected components.
+
+A visual connector establishes only a visual connection between the components, depicting an intent to connect. No interaction between the components is established. If the target node is not an Operator-backed service the *Create a visual connector* tool-tip is displayed when you drag an arrow to a target node.
+
+== Creating a visual connection between components
+
+You can depict an intent to connect application components using the visual connector.
+
+This procedure walks through an example of creating a visual connection between a PostgreSQL Database service and a Spring PetClinic sample application.
+
+.Prerequisites
+
+* Ensure that you have created and deployed a Spring PetClinic sample application using the *Developer* perspective.
+* Ensure that you have created and deployed a Crunchy PostgreSQL database instance using the *Developer* perspective.
+
+.Procedure
+
+. Hover over the Spring PetClinic sample application to see a dangling arrow on the node.
+. Click and drag the arrow towards the `hippo-pgbouncer` Deployment to connect the Spring PetClinic sample application with it.
+. Click on the `spring-petclinic-rest` Deployment to see the *Overview* panel. Under the *Details* tab, click the edit icon in the *Annotations* section to see the *Key = `app.openshift.io/connects-to`* and *Value = `[{"apiVersion":"apps/v1","kind":"Deployment","name":"hippo-pgbouncer"}]`* annotation added to the Deployment.
+
+Similarly you can create other applications and components and establish visual connections between them.
+
+
+== Creating a binding connection between components
+
+[NOTE]
+====
+Currently, a few specific Operators like the *PostgresSQL Database* Operator's service instances are bindable.
```

Comment:
there are multiple PostgresSQL operator available, which one do you mean exactly?

Reply:
@pedjak @pmacik We need those inputs from you. Kindly provide.

Reply:
@pmacik please correct me, but I think that currently there is no operator that can be used properly from ODC?

Reply:
@pedjak The Crunchy Postgresql as shown and used in quick-start can be used in ODC.

Reply:
@Srivaralakshmi correct me if I'm wrong, but I believe that the reference in text to `PostgreSQL Database Operator` means "that operator that we installed in Prerequisities", it's not the actual name of the operator.

Reply:
As far as I know the only PostgreSQL DB operators supported by SBO at GA are [Crunchy PostgreSQL Operator](https://operatorhub.io/operator/postgresql) and [Cloud Native PostgreSQL](https://operatorhub.io/operator/cloud-native-postgresql).
@pedjak please confirm

Reply:
However, I'm not sure if tha Cloud Native one is supported by ODC itself